wagtail/documents/templates/wagtaildocs/homepage/site_summary_documents.html

```diff
@@ -3,9 +3,9 @@
 <li class="icon icon-doc-full-inverse">
     <a href="{% url 'wagtaildocs:index' %}">
     {% blocktrans count counter=total_docs with total_docs|intcomma as total %}
-        <span>{{ total }}</span> Document
+        <span>{{ total }}</span> Document <span class="visuallyhidden">created in {{ site_name }}</span>
```

Comment:
"created in" needs to be translatable in all templates, so `{% trans "created in" %}

Reply:
This is inside a `{% blocktrans %}` tag, so that's already covered. (Not sure it's a good thing to have so much HTML inside a string to be passed to translators, but I can't immediately see a good way around it...)

Reply:
🤦‍♂ I missed the blocktrans. Sorry for the noise!